shared/src/main/scala/coop/rchain/metrics/Metrics.scala

```diff
@@ -68,4 +69,43 @@ object Metrics extends MetricsInstances {
   val BaseSource: Source                           = Source("rchain")
 }
 
-sealed abstract class MetricsInstances {}
+sealed abstract class MetricsInstances {
+  def readerTMetrics[F[_], E](m: Metrics[F]): Metrics[ReaderT[F, E, ?]] =
```

Comment:
I think this should be a one-liner using `mapK[F ~> ReaderT[F, E, ?]]`

Reply:
+1

Reply:
Metrics has 
```
def timer[A](name: String, block: F[A])
```
which requires a 
```
envToTask: TaskEnv ~> Task = λ[TaskEnv ~> Task](_.run(NodeCallCtx.init))
```
to be handled properly

which do you prefer?